slides/slides.md

```diff
@@ -136,6 +136,8 @@ interface Global {
 
   // + properties copied from globalThis filtered by keys
 }
+
+const newGlobal = new globalThis.Global();
```

Comment:
I discovered during a dry-run that having this on screen while explaining the interface would be useful.